test/var_compatibility_test/effr_diag.meta

```diff
@@ -41,6 +41,13 @@
   kind = kind_phys
   intent = in
   top_at_one = True
+[ scalar_var ]
```

Comment:
This is probably my capgen ignorance showing, but I'm not seeing how these new test variables are checking the new logic. They aren't in DDTs and they aren't duplicate names with different standard names, unless I'm missing something?.

Reply:
@mkavulich it's the variable naming in the caps. The test now uses  same local_name for 3 different standard_names. To see this, run the var_compatabilty_test and look at the suite cap in ct_build/ccpp, you will see that each instance of scalar_var in the Cap has been renamed to a unique name, scalar_var1, Scalar_var2, and Scalar_var3, respectively.

Reply:
@dustinswales another testing question!

I'm wondering if we need to amend the test to perform some sort of math on these scalar variables to validate the local name issue. The test as-is (which is great!) tests that it doesn't break everything to have multiple instances of a local name with the same standard name. Do we also need to test that the framework is passing in the CORRECT local name to the schemes?

Reply:
@peverwhee Good idea. Done.